index.js

```diff
@@ -5,7 +5,7 @@ function loader (registryOrVersion) {
     constructor (type, count, metadata, nbt) {
       if (type == null) return
 
-      if (metadata instanceof Object && metadata !== null) {
+      if (metadata instanceof Object) {
```

Comment:
Why

Reply:
Because `null instanceof Object` is false, so that's doubled code 

![image](https://user-images.githubusercontent.com/69903408/228068860-5e12ede4-8654-4618-b9ef-6af1a3c163e7.png)

Reply:
@rom1504 any updates on the review, especially the exported types, would be useful?